repository/flink/docs/demo/tests/flink-demo/02-dependency-crs.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install zookeeper --operator-version=0.3.0 --skip-instance
```

Comment:
Let's use local packages here, i.e. relative paths instead of packages from the community repository. This will allow us to see if (local) changes to any of these operators break this test.
Here and for `kafka` and `flink` below.

Reply:
perhaps these tests are different... and should be relocated or not hooked into the infra... this is under the "demo" folder and these teams are similar to our use of kuttl for MWT.. they setup a cluster with the flink demo.

On the other hand... I could see value in having the demo checked on PRs to confirm it still works... hmm..